sigle/src/modules/layout/components/AppHeader.tsx

```diff
@@ -90,6 +133,41 @@ export const AppHeader = () => {
           </Box>
         </Link>
       </Flex>
+
+      <Flex gap="2">
+        <Flex
+          css={{
+            display: 'flex',
+            '@xl': {
```

Comment:
@gregogun I am a bit confused with the breakpoints, but shouldn't this be `md`?

Reply:
ahh yeah that's correct it should be `md`, updated 👍